source/module_io/numerical_basis.cpp

```diff
@@ -694,8 +694,9 @@ void Numerical_Basis::output_overlap_Q(std::ofstream& ofs,
             const int dim = Qtmp.getSize();
             for (int i=0; i<dim; i++)
             {
-                if ( count%4==0 ) ofs << "\n";
+                if ( count%4==0 ) ofs << std::endl;
                 ofs << " " << Qtmp.ptr[i].real() << " " << Qtmp.ptr[i].imag();
+                ofs.flush();
```

Comment:
Will high frequency flushing slow down the efficiency of this function?

Reply:
I think so, but I notice after every four numbers are written, there will be a flush by endl, I have deleted flush here